desafio-02/paullomarttins/python/desafio-02.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+"""
+Desafio 02, listando números primos de 1 até 10000!
+"""
+def num_primo(num):
+    for valor in range(2, num):
```

Comment:
Este bloco de `for` funciona perfeitamente, mas sugiro depois dar uma lida [nas duas primeiras linhas desta seção do artigo de números primos na Wikpedia](https://pt.wikipedia.org/wiki/N%C3%BAmero_primo#Os_%C3%A1tomos_da_aritm%C3%A9tica) e tentar aplicar uma otimização para que este loop não precise percorrer **todos** os números entre 2 e o número que quer testar subtraido de 1